README.md

```diff
@@ -4,14 +4,20 @@ Write your answer below each question.
 
 ### Question 1
 (a) What is a click handler? 
+A click handler is a function that is invoked at the specified event, in this case, a click on the jquery object. You can use .on('click', andPassTheFunctionYouWant), or .click(). 
 
 (b) Where in your JS file should you put it, and WHY? 
+You put it inside the document-ready function so that it can occur after any dependencies on other code is ready. However, you can write the function you will want to call with your event wherever you want (likely in the app namespace) and pass that function along with the click handler - you wouldn't want to do on.('click', function(){}), cause it's sloppy and not reusable, although I suppose you could.
+
+As a somewhat related question, many of my questions these days have to do with how best and most intuitively to organize your code throughout the whole app.
```

Comment:
Definitely very valid questions. I suspect that the most satisfying answer will come when we learn Angular.